index.js

```diff
@@ -2,6 +2,8 @@
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
+toggleColor(document.querySelector("h1#header"))
```

Comment:
All we need to do is add an event listener to that header. We provide a callback function to be executed when our "click" event gets triggered.

The callback function we can pass in to the event listener is already made for us!. We just need to pass in the event target as an argument to the function.
```
header.addEventListener("click", function(e){
    toggleColor(e.target)
})
```
With ES6 notation
```
header.addEventListener("click", e => toggleColor(e.target))
```